src/AuthContext.js

```diff
@@ -8,12 +8,13 @@ export const AuthProvider = ({ children }) => {
 
   const login = (email) => {
     setIsAuthenticated(true);
-    setUsername(email); // Utilisation de l'email comme nom d'utilisateur
+    setUsername(email);
   };
 
   const logout = () => {
     setIsAuthenticated(false);
     setUsername(null);
+    localStorage.removeItem('session_token'); // Ensure the session token is removed
```

Comment:
Do you really need that additional comment ?

The method is named "removeItem" and the comment says "ensure that it removes the token".

It's creating noise in the readability of the code as it states the obvious.